crates/router/src/connector/adyen/transformers.rs

```diff
@@ -4208,8 +4231,49 @@ impl<F> TryFrom<&AdyenRouterData<&types::PayoutsRouterData<F>>> for AdyenPayoutC
                     recurring: RecurringContract {
                         contract: Contract::Payout,
                     },
+                    selected_brand: None,
                     merchant_account,
-                    bank: bank_details,
+                    additional_data: None,
+                    bank: Some(bank_details),
+                    reference: item.router_data.request.payout_id.to_owned(),
+                    shopper_reference: item.router_data.merchant_id.to_owned(),
+                    shopper_email: customer_email,
+                    shopper_name: ShopperName {
+                        first_name: address.get_first_name().ok().cloned(),
```

Comment:
Don't we need to throw missing field error in case of first_name and last_name none?

Reply:
Since it is a mandatory field, error propagation is enabled now